std/StringTools.hx

```diff
@@ -598,6 +598,25 @@ class StringTools {
 		}
 	}
 
+	/**
+		This is a convenience function for `ereg.replace(str, by)`.
+
+		See `EReg.replace` for implementation details.  
+		@see https://haxe.org/manual/std-regex-replace.html
+	**/
+	public static inline function eregReplace(str : String, ereg : EReg, by : String ) : String {
+		return ereg.replace(str, by);
+	}
+
+	/**
+		Returns `true` if `str` contains `value` and returns `false` otherwise.
+		Returns `false` if `str` is `null`.
+		When `value` is `null`, the result is undefined.
+	**/
+	public static inline function contains(str : String, value : String) : Bool {
+		return if (str == null) false else str.indexOf(value) != -1;
```

Comment:
simplifying:
```haxe
return str != null && str.indexOf(value) != -1;
```

Reply:
I'm not sure that we even want to specify `str == null` behaviour here, especially given that `StringTools` is meant to be used as a static extension, and I think it's weird that `s.contains(...)` wouldn't fail if `s` is `null`.

But if we do, the type hint should be `Null<String>`.